pkgs/tools/security/sudolikeaboss/default.nix

```diff
@@ -19,9 +19,11 @@ buildGoPackage rec {
 
   propagatedBuildInputs = with darwin.apple_sdk.frameworks; [
     Cocoa
-    fixDarwinDylibNames
   ];
 
+  propagatedNativeBuildInputs =
```

Comment:
This should never be propagated, that makes it impossible to build libraries rpath or executable relative references when this is a dependency.